src/AudioUnit/AUGraph.cs

```diff
@@ -183,8 +218,18 @@ public AudioUnitStatus RemoveRenderNotify (RenderDelegate callback)
 
 			if (renderers.Count != 0) {
 				using (var buffers = new AudioBuffers (_ioData)) {
+#if NET
+					foreach (RenderDelegate renderer in renderers) {
+						var tempActionFlags = *_ioActionFlags;
+						var tempTimeStamp = *_inTimeStamp;
+						renderer (tempActionFlags, tempTimeStamp, _inBusNumber, _inNumberFrames, buffers);
```

Comment:
The temporary values here are unnecessary, because due to an incorrect binding, the ioActionFlags and inTimeStamp values aren't updated by the renderer callback:

```suggestion
						renderer (*_ioActionFlags, *_inTimeStamp, _inBusNumber, _inNumberFrames, buffers);
```

These arguments should be `ref` arguments in the `RenderDelegate`: https://github.com/xamarin/xamarin-macios/blob/4c36652927ceb4098472a913b0e579000294d4c2/src/AudioUnit/AudioUnit.cs#L132

But that would be a breaking change, so we can't do that (we'd have to add (and correct) API instead of changing the existing one).

In any case, that's completely out of scope for this PR. I'd just make the .NET code just as broken as the non-.NET code, and maybe we'll get around to fixing it properly one day.